### PR TITLE
Grid-world update

### PR DESCRIPTION
**Pull request by BoZenKhaa**
Update to the code and the text of the gridworld tutorial notebook to work with the current POMDPs.jl packages.

For MCTS to work, I had to bump the compat of POMDPs in `Project.toml` from 0.8 to 0.9. I hope that doesn't break the other notebooks. 

The testset fails for all the notebooks on my machine, I am not sure whether I am running it properly, the errors I get look like this:
```
ERROR: syntax: invalid escape sequence
Stacktrace:
 [1] top-level scope at none:0
notebooks: Test Failed at POMDPExamples.jl\test\runtests.jl:14
  Expression: success(proc)
```

As a sidenote: I was using the AIMA gridworld example when using the notebook. I like it better as it is much smaller and doesn't have negative reward for bumping into walls, but I have kept the example from [artint.info](https://artint.info/2e/html/ArtInt2e.Ch9.S5.html) in this pull request. In the notebook, the examle from artint.info is implemented without the negative reward for wall bumpin, which causes an issue, as the policy from VI doesn't fit with the policy presented in the included image. But I think that's allright as afterall this is not a notebook to teach MDPs.

### CODE REVIEW

**Comment by zsunberg** [comment]
Wow @BoZenKhaa , this is great! Thank you! I will have to look at it more closely tomorrow and figure out what the test failure means, but we should be able to merge it soon.

File: Project.toml [hunk 1]
```diff
@@ -4,20 +4,22 @@ authors = ["Zachary Sunberg <sunbergzach@gmail.com>"]
...
 BeliefUpdaters = "8bb6e9a1-7d73-552c-a44a-e5dc5634aac4"
-FIB = "13b007ba-0ca8-5af2-9adf-bc6a6301e25a"
+D3Trees = "e3df1716-f71e-5df9-9e2d-98e193103c45"
+DiscreteValueIteration = "4b033969-44f6-5439-a48b-c11fa3648068"
+MCTS = "e12ccd36-dcad-5f33-8774-9175229e7b33"
 POMDPModelTools = "08074719-1b2a-587c-a292-00f91cc44415"
 POMDPModels = "355abbd5-f08e-5560-ac9e-8b5f2592a0ca"
 POMDPPolicies = "182e52fb-cfd0-5e46-8c26-fd0667c990f4"
 POMDPSimulators = "e0d0a172-29c6-5d4e-96d0-f262df5d01fd"
+POMDPToolbox = "0729bffe-8e6b-52fa-a3fa-893719b744f4"
```
Comment (rejuvyesh) [inline Project.toml:15]: This probably shouldn't be here.

Comment (BoZenKhaa) [reply Project.toml]: I agree, I accidentally left it in after playing with the legacy notebook. I removed it in the next commit.

**Review by BoZenKhaa — COMMENTED** [review]
I noticed I removed some dependencies needed in other notebooks, here I am fixing that. Also adding NBInclude that is used in the testing script, I am not sure whether that should go here.

**Comment by BoZenKhaa** [comment]
I think the reason the tests were failing on my win machine was because the backslashes in the notebook path were being treated as escapes for the characters in the path string. As per [this topic on discourse](https://discourse.julialang.org/t/invalid-escape-sequence/12154/3), using raw paths fixed that for me. So with the latest commit, the tests are passing on my machine.

**Comment by zsunberg** [comment]
Ok, looks good now! Thanks again for this update @BoZenKhaa it is much appreciated!

**Comment by BoZenKhaa** [comment]
Thanks! I needed a MDP tutorial and this was the easiest way to get something good looking ;-)

**Pull request merged**